content/blog/tune-2/index.Rmd

```diff
@@ -0,0 +1,166 @@
+author: Max Kuhn, Simon Couch, Emil Hvitfeldt, Hannah Frick
+description: >
+    A new version of tune brings significant updates for model tuning and 
+    parallel processing. 
+
+photo:
+  url: https://unsplash.com/photos/black-car-stereo-turned-on-at-7-qHbg3DKB1Y0
+  author: Stephen Andrews
+
+# one of: "deep-dive", "learn", "package", "programming", "roundup", or "other"
+categories: [package] 
+tags: [tidymodels,postprocessing,parallel processing]
+---
+
+```{r}
+#| label: startup
+#| include: false
+
+library(tidymodels)
+theme_set(theme_bw())
+options(cli.width = 72, cli.ansi = FALSE)
+```
+
+We're very chuffed to announce the release of [tune](https://tune.tidymodels.org) **2.0.0**. tune is a package that can be used to resample models and/or optimize their tuning parameters 
+
+You can install it from CRAN with:
+
+```{r, eval = FALSE}
+install.packages("tune")
+```
+
+This blog post will describe the two major updates to the package. You can see a full list of changes in the [release notes](https://tune.tidymodels.org/news/index.html#tune-200)
+
+Those two big improvements to the package: new parallel processing features and postprocessing. 
+
+## Using future or mirai for parallel processing
+
+[Historically](https://www.tidyverse.org/blog/2024/04/tune-1-2-0/#modernized-support-for-parallel-processing), we've used the foreach package to run calculations in parallel. Sadly, that package is no longer under active development, so we've removed support for it and added functionality for the [future](https://future.futureverse.org/) and [mirai](https://mirai.r-lib.org/) packages. 
+
+Previously, you would load a foreach parallel backend package, such as doParallel, doMC, or doFuture, and then register it. For  example:
+
+```
+library(doParallel)
+cl <- makePSOCKcluster()
+registerDoParallel(cl)
+```
+
+Instead, you can use the future package via:
+
+```
+library(future)
+plan("multisession")
+```
+
+or the mirai package by using
+
+```
+library(mirai)
+daemons(num_cores)
+```
+
+Each of these is configurable to run in various ways, such as on remote servers. 
+
+## Tuning your postprocessor
+
+A postprocessor is an operation that modifies model predictions.  For example, if your classifier can separate classes but its probability estimates are not accurate enough, you can add a _calibrator_ operation that can attempt to adjust those values. Another good example is for binary classifiers, where the default threshold for classifying a prediction as an event can be adjusted based on its corresponding probability estimate. 
+
+Currently, we've enabled postprocessing using the [tailor package](https://www.tidyverse.org/blog/2024/10/postprocessing-preview/). The operations that are currently available: 
+
+ - `adjust_numeric_calibration()`: Estimate and apply a calibration model for regression problems. 
+ - `adjust_numeric_range()`: Truncate the range of predictions.
+ - `adjust_predictions_custom()`: A general `mutate()`-like adjustment. 
```

Comment:
I think we should move `adjust_predictions_custom()` to the end of the list. this way it gives a little more structure to the list itself